abdiff/core/calc_ab_metrics.py

```diff
@@ -238,12 +239,12 @@ def _get_source_counts(
     sources: list[str],
     analysis: dict,
 ) -> dict:
-    any_field_modified_condition = " OR ".join(f"{field} = 1" for field in fields)
     for source in sources:
         total_count = conn.execute(
             f"""
             select count(*) from record_diff_matrix
-            where source = '{source}' and ({any_field_modified_condition})
+            where source = '{source}'
+            and has_diff;
```

Comment:
Same question here re: whether query should instead read:

```sql
has_diff = 1
```

Reply:
Same answer!